src/Symfony/Component/Validator/Constraints/All.php

```diff
@@ -16,23 +16,12 @@
  * @Target({"PROPERTY", "METHOD", "ANNOTATION"})
  *
  * @author Bernhard Schussek <bschussek@gmail.com>
+ *
+ * @api
```

Comment:
should be removed. We dropped such tag years ago in favor of `@internal` and `@experimental`